beacon-chain/rpc/eth/node/handlers.go

```diff
@@ -37,14 +38,29 @@ func (s *Server) GetSyncStatus(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	isElOnline := s.ExecutionChainInfoFetcher.ExecutionClientConnected()
+	if !isElOnline {
+		isSyncing, err := s.ExecutionChainInfoFetcher.IsExecutionClientSyncing(ctx)
```

Comment:
Yes but this is just an api call at the time. When that happens it either returns an error while syncing or it's not connected and not syncing because of an issue that I noted in the pr description.

Reply:
makes sense 👍